src/main/java/com/HospitalPatientInformation.java

```diff
@@ -0,0 +1,25 @@
+package com;
+
+public class HospitalPatientInformation {
+    private String name;
```

Comment:
Those variables are injected only by constructor so you may define them as a final (concept of immutability)
https://docs.oracle.com/javase/tutorial/essential/concurrency/imstrat.html
http://pelikan-i-java.blogspot.com/2014/08/niemodyfikowalne-obiekty.html